### ISSUE TITLE
Provide accessibility 

### ISSUE THREAD

**Issue by leandromoreira** [clappr/clappr#863]
This is a huge topic, but we should at least provide minimal support for: **play**, **pause** and **mute**  we can start with the basic [aria-label](https://www.w3.org/TR/wai-aria/states_and_properties#aria-label) 

Thanks to @bernardocamilo

**Comment by leandromoreira** [comment]
For the SVGS https://www.sitepoint.com/tips-accessible-svg/

**Comment by leandromoreira** [comment]
An real example https://www.paypal-engineering.com/2014/09/05/introducing-an-accessible-html5-video-player/

**Comment by dovematrix** [comment]
Has there been any progress on this issue?  The WAVE extension, http://wave.webaim.org/extension/, for the Google Chrome browser reports three accessibility errors.  The play/pause, full screen, and another button does not have descriptive text.

**Comment by leandromoreira** [comment]
@dovematrix unfortunately, I don't think so 😞

**Comment by cinnabot** [comment]
Hi, I have some suggestions to help solving this issue!



Here is a list of similar issues:


1. [Update HLS Provider used by HLS plugin](https://github.com/clappr/clappr/issues/54)

2. [HD label is not working](https://github.com/clappr/clappr/issues/130)

3. [Error with hlsjs when providing source in load method instead of options object](https://github.com/clappr/clappr/issues/589)

4. [Chromeless player should not play/pause when clicking over the video](https://github.com/clappr/clappr/issues/290)

5. [Implement the new UI](https://github.com/clappr/clappr/issues/132)






Hope this is useful!

Cheers,
Cinnabot.


PS: Liked my suggestions? Install me in your repository! Visit: [getcinnamon.io](https://www.getcinnamon.io).

**Comment by bikegriffith** [comment]
Just submitted a PR to at least appease WAVE with some simple aria-labels
